docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/index.md

```diff
@@ -19,38 +19,21 @@ We've open-sourced the [OctoTFS repository in GitHub](https://github.com/Octopu
 
 ## Installing the extension
 
-If you're using **Azure DevOps or on-premises Team Foundation Server (TFS) 2017 (or newer)** you can simply [install the extension from the marketplace](https://marketplace.visualstudio.com/items/octopusdeploy.octopus-deploy-build-release-tasks) and follow the instructions below.
+You can simply [install the extension from the marketplace](https://marketplace.visualstudio.com/items/octopusdeploy.octopus-deploy-build-release-tasks) and follow the instructions below.
 
 If you're using an earlier version of TFS, see the [Extension Compatibility documentation](extension-compatibility.md) for details on where to get a compatible extension.
```

Comment:
TFS => Azure DevOps

Reply:
I think in this context akes sense to call it TFS?

Reply:
I have updated this section to provide more context